src/BenchmarkDotNet/Detectors/Cpu/Windows/MosCpuDetector.cs

```diff
@@ -41,14 +43,23 @@ public bool IsApplicable() => OsDetector.IsWindows() &&
                     processorsCount++;
                     physicalCoreCount += (int)(uint)moProcessor[WmicCpuInfoKeyNames.NumberOfCores];
                     logicalCoreCount += (int)(uint)moProcessor[WmicCpuInfoKeyNames.NumberOfLogicalProcessors];
-                    sumMaxFrequency = (int)(uint)moProcessor[WmicCpuInfoKeyNames.MaxClockSpeed];
+                    double tempMaxFrequency = Convert.ToDouble((uint)moProcessor[WmicCpuInfoKeyNames.MaxClockSpeed]);
```

Comment:
Why is `Convert` necessary here? Isn't `uint` implicitly convertible to `double`?